### PR TITLE
LRDOCS-8385 Configuring SEO and OpenGraph 

### PR DESCRIPTION
**Pull request by Alec-Shay**
https://issues.liferay.com/browse/LRDOCS-8385
See both:
- https://issues.liferay.com/browse/LPS-115404, and
- https://issues.liferay.com/browse/LPS-120333

Originally from @JamesAGarcia at: https://github.com/Alec-Shay/liferay-learn/pull/44

This PR includes:
- A new article, "Configuring SEO and Open Graph"
- Extensive edits to "Creating a Display Page Template"
- Other minor edits within the same section

Please let us know your thoughts when you get the chance. Thanks!

### CODE REVIEW

File: docs/dxp/7.x/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+* [Configuring SEO Settings](#configuring-seo-settings)
+* [Configuring Open Graph Settings](#configuring-open-graph-settings)
+
+## Configuring SEO Settings
+
+Search engine optimization (SEO) refers to the methods used to improve your Page's ranking in a search engine results page (SERP). With Liferay DXP, you can create Display Page templates that dynamically configure your displayed content for SEO.
+
+### HTML Title
+
+The *HTML Title* field defines a Display Page's title `<meta>` tag. This title is used by search engines to rank your Page and serves as the Page's heading in search engine results. By default, Display Page templates map this field to *Title*, but you can select any other Text field. The recommended length for an HTML title is under 60 characters.
```
Comment (boton) [inline docs/dxp/7.x/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md:20]: > The *HTML Title* field defines a Display Page's title `<meta>` tag.

The HTML title defines the Display Page's `<title>` tag

**Review by boton — COMMENTED** [review]
Everything about SEO is fine except for a small comment on the HTML title.

File: docs/dxp/7.x/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md (hunk 1 above)
Comment (Alec-Shay) [reply docs/dxp/7.x/en/site-building/displaying-content/using-display-page-templates/configuring-seo-and-open-graph.md]: Pushed up a new commit with this correction.

**Comment by sez11a** [comment]
Just started reviewing :)

**Comment by sez11a** [comment]
Superseded by https://github.com/jrhoun/liferay-learn/pull/694

**Pull request closed without merging**